composer.json

```diff
@@ -21,12 +21,12 @@
     ],
     "require": {
         "php": "^7.1",
-        "sonata-project/admin-bundle": "^3.35",
-        "sonata-project/block-bundle": "^3.11",
+        "sonata-project/admin-bundle": "^3.45",
+        "sonata-project/block-bundle": "^3.14",
         "sonata-project/cache-bundle": "^2.1.7",
-        "sonata-project/core-bundle": "^3.9",
+        "sonata-project/core-bundle": "^3.14",
```

Comment:
Travis crashes because of the conflict with exporter, try maybe with `^3.13.7`, it does not have that conflict.